com.github.KRTirtho.Spotube.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Nitpick: ideally this file should be maintained in your repo and included in Spotube-linux-x86_64.tar.xz